doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -660,6 +660,9 @@ The following changes are recommended for your application to work optimally aft
 
 * For LwM2M applications, replace the :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_DTLS_CID` Kconfig option with :kconfig:option:`CONFIG_LWM2M_DTLS_CID`.
 
+* The :ref:`nrf53_audio_app` have changed the default controller from the :ref:`lib_bt_ll_acs_nrf53_readme` to Nordic Semiconductor's :ref:`ug_ble_controller_softdevice` (:ref:`softdevice_controller_iso`). Make sure to transition to using the SoftDevice Controller. There should be no negative impact on performance for the nRF5340 Audio applications. This change will enable use of standard tools for building and configuring the controller. The SoftDevice Controller will for some time not have a QDID for the ISO operation.
```

Comment:
```suggestion
* The :ref:`nrf53_audio_app` have changed the default controller from the :ref:`lib_bt_ll_acs_nrf53_readme` to Nordic Semiconductor's :ref:`ug_ble_controller_softdevice` (:ref:`softdevice_controller_iso`).
  The new controller is included and built automatically.
  Make sure to remove references to LE Audio controller for nRF5340 from your application.

  There should be no negative impact on performance of the nRF5340 Audio applications with the new controller.  
  This change enables the use of standard |NCS| tools and procedures for building and configuring the controller. 
  The SoftDevice Controller will for some time not have a QDID for the ISO operation.
```
To match one sentence per line rule and provide direct info to the user about what to do with the old controller.

Reply:
@koffes , this edit has not made it to your latest push despite your +1.